device/common/profiles/th2/7260/RDMA-CENTRIC/buffers_defaults_t1.j2

```diff
@@ -51,11 +51,8 @@
 {%- endmacro %}
 
 {% import 'buffers_extra_queues.j2' as defs1 with context %}
-{% import 'buffers_extra_pgs.j2' as defs2 with context %}
```

Comment:
qos.json.j2 under RDMA-CENTRIC needs to be updated as well

Reply:
Thanks for catching this error. I did a global search again, confirmed all required files are updated. Please take a look one more time.